src/components/SingleStock.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getStockDetail } from '../redux/stocks/stocksSlice';
+import '../css/SingleStock.css';
+import demologo from '../assets/demologo2.png';
+
+const SingleStock = () => {
+  const { stockDetail } = useSelector((store) => store.stocks);
+  const { loadingDetail } = useSelector((store) => store.stocks);
+  const [imgLoadError, setImgLoadError] = useState(false);
+
+  const handleImgLoad = () => {
+    console.log('This is printing');
+    setImgLoadError(true);
+    console.log('imgLoadError: ', imgLoadError);
+  };
+
+  const { symbol } = useParams();
+  console.log('symbol: ', symbol);
+
+  const singleStock = stockDetail;
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getStockDetail(symbol));
+  }, [dispatch]);
+
+  if (loadingDetail) {
+    return (
+      <div data-testid="stockDetail">
+        Loading...please wait
+        <Link to="/">
+          <button type="button">Go back</button>
+        </Link>
+      </div>
+    );
+  }
+
+  return (
+    <div className="single-stock" data-testid="stockDetail">
+      <div className="overhead">
+        <img
+          src={(imgLoadError)
+            ? demologo : singleStock.image}
+          alt="logo"
+          className="logo"
+          onError={handleImgLoad}
+        />
+        <p className="overhead-symbol">
+          &copy;
+          {symbol}
+        </p>
+      </div>
```

Comment:
👍 You're on the right track with the details stock page! To make it even closer to the design and improve the visual separation of the two parts, consider the following suggestions:
- [ ] To align your project more closely with the design and enhance the top part of the details stock page (`overhead`), kindly consider the following:
1. **Background Color:** In the design, the top part (`overhead`) has a distinct background color that differs from the bottom part (`info-container`) and navbar. To create a clear visual separation, update the background color of the `overhead` section to match the design specifications.
2. **Layout and Content:** In the design, there's a logo or image on the left side of the `overhead` section, you also have a image in the left side. while the right side contains the company name and some numeric values about the company. To achieve a closer match to the design, arrange your elements accordingly within the `overhead` div.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/AliRezaBmeDu/metrics-webapp/assets/55840999/b982928e-af05-423b-b12e-23ec8c5b53fd)| ![image](https://github.com/AliRezaBmeDu/metrics-webapp/assets/55840999/0b70fb22-dcdc-4cd1-b370-44d76331058d)|

By implementing these adjustments, you'll ensure that the top part of your details stock page closely resembles the design and offers a more visually appealing and structured user interface. 🎨📊 Keep up the great work! 👏💼

Reply:
done